views/add-event.hbs

```diff
@@ -0,0 +1,50 @@
+<div class="add-eventContainer">
+  <h3 class="userAddEvent">{user name} please add event</h3>
+
+  <form class="add-event-form" action="/add-event" method="post">
+    <label>Title: </label>
+    <input class="form-control" type="text" name="name" placeholder="title" required>
+
+    <label>Picture: </label>
+    <input class="form-control" type="file" name="imageUrl" accept="image/*" placeholder="choose picture">
+
+    <label>Start Time: </label>
+    <input class="form-control" type="text" name="startTime" placeholder="start time" required>
+
+    <label>End Time: </label>
+    <input class="form-control" type="text" name="endTime" placeholder="end time" required>
+
+    <label>Location: </label>
+    <input class="form-control" type="text" name="place" placeholder="place" required>
+
+    <label class="sel1">Category: </label>
+    <select class="options form-control" name="categories" required>
+      <option disabled selected>Select your option..</option>
+      <option value="music">Music</option>
+      <option value="educational">Educational</option>
+      <option value="theatrical">Theatrical</option>
+      <option value="social Gathering">Social Gathering</option>
+      <option value="cinema">Cinema</option>
+      <option value="sport">Sport</option>
+      <option value="cultural Event">Cultural</option>
+      <option value="guesthouse">Guesthouse</option>
+      <option value="store">Store</option>
+      <option value="restaurant">Restaurant</option>
+      <option value="historical site">Historical site</option>
+      <option value="hangout Spots">Hangout Spots</option>
+      <option value="miscellaneous">Miscellaneous</option>
+      <option value="unspecified">Unspecified</option>
```

Comment:
some of these options look like they are not really for `events`, but for `places`

Reply:
you should add your suggestions for event categories in the issue on the platform:
https://github.com/foundersandcoders/open-tourism-platform/issues/82